components/Datepicker/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@gemeente-denhaag/pickersutilsprovider",
```

Comment:
Do these old picker packages need to be deprecated from npm?

Reply:
Already did that 👍🏼